src/test/java/com/sonymobile/tools/gerrit/gerritevents/dto/GerritEventTypeTest.java

```diff
@@ -0,0 +1,17 @@
+package com.sonymobile.tools.gerrit.gerritevents.dto;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class GerritEventTypeTest {
+    
+    @Test
+    public void testSetInteresting() {
+        GerritEventType type = GerritEventType.CHANGE_MERGED;
+        type.setInteresting(false);
```

Comment:
I'm not sure at the moment how static changes affects other tests. Some engines like newer surefire runs each suite in it's own vm while others run them in the same, so if something goes wrong after line 12 it might still be set to `false`.

Perhaps wrap line 12 to 15 in a `try {} finally { type.setInteresting(true); } just in case?

Reply:
Done.